src/ethereum_test_specs/blockchain.py

```diff
@@ -652,7 +654,7 @@ def make_fixture(
             blocks=fixture_blocks,
             last_block_hash=head,
             pre=pre,
-            post_state=alloc,
+            post_state=alloc if not self.exclude_post_state else {},
```

Comment:
Initially, I did it with None instead of {}. This means the field didn't exist in the output.

While Geth was okay with it, Reth now is a bit more strict and find that problematic since it always expects the field to be set.

So the less invasive way is always have the field, but empty.

Reply:
I think having the field as `{}` is a bit misleading, as compared to removing it.

I feel like there should be some logic in the client consumer that acknowledges the field is missing and then proceeds to only check the state root.

Cc'ing @danceratopz since he has reached out to reth team in the past and they have been responsive on these changes to the test formats iirc.

Reply:
Sharing here the relevant current reth logic about this:
![image](https://github.com/user-attachments/assets/b1330e43-42bd-4a0d-9ce9-ba5815b2f9aa)

cc @kevaundray  since he was around this code some days ago!

Reply:
Happy to make this change!

Since it doesn't change any of the tests, it would mainly be a case of: not returning an error if the post state is missing and just returning true.

Reply:
@marioevz, I talked with @kevaundray and he'll do the reth change in a fork that we can use for a while before upstreaming the change. 

So, I'll switch to use `None` instead as you prefered. Whenever the reth fork is ready I'll test again just in case, and I can ping you again to potentially merge the pr.

Reply:
@marioevz, done:
- Did the change to `None` here.
- Tested in the new with the new https://github.com/eth-applied-research-group/zkevm-benchmark-workload Reth fork with the fix, and is all good.